src/modules/views/OnlyOffice/Title.jsx

```diff
@@ -21,25 +21,19 @@ const useStyles = makeStyles(theme => ({
 
 const Title = () => {
   const { isMobile } = useBreakpoints()
-  const {
-    fileId,
-    isPublic,
-    isFromSharing,
-    isInSharedFolder,
-    isEditorModeView,
-    isTrashed
-  } = useOnlyOfficeContext()
+  const { fileId, isPublic, isEditorModeView, isTrashed } =
+    useOnlyOfficeContext()
+  const sharingInfos = useSharingInfos()
+  const { loading, isSharingShortcutCreated } = sharingInfos
   const styles = useStyles()
 
-  const showBanner = useMemo(
-    () =>
-      showSharingBanner({
-        isPublic,
-        isFromSharing,
-        isInSharedFolder
-      }),
-    [isPublic, isFromSharing, isInSharedFolder]
-  )
+  // Check if the sharing shortcut has already been created (but not synced)
+  const isShareAlreadyAdded = !loading && isSharingShortcutCreated
+  // Check if you are sharing Cozy to Cozy (Link sharing is on the `/public` route)
+  const isPreview = window.location.pathname === '/preview'
```

Comment:
Why do we talk about /public and use /preview ?

Reply:
Sharing from Cozy to Cozy public is possible via the `/preview` route.